src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
```

Comment:
Could be, what's the guideline here? My reasoning: PRO requires CORE, CORE requires Common -> have this available in Pro.